packages/flex-plugin/src/lib/__tests__/flex-plugin.test.ts

```diff
@@ -36,12 +61,27 @@ describe('@twilio/flex-plugin', () => {
             }
         }
 
+        it('should test injected properties', () => {
+            const plugin = new TestPlugin();
+
+            expect(plugin.name).toEqual(PLUGIN_NAME);
+            expect(plugin.version).toEqual(PLUGIN_VERSION);
+            expect(plugin.identifier).toEqual(IDENTIFIER);
+            expect(plugin.dependencies['flex-plugin-scripts']).toEqual(FLEX_PLUGIN_SCRIPTS_VERSION);
+            expect(plugin.dependencies['flex-plugin']).toEqual(FLEX_PLUGIN_VERSION);
+            expect(plugin.dependencies['craco-config-flex-plugin']).toEqual(CRACO_CONFIG_VERSION);
+            expect(plugin.dependencies['flex-ui']).toEqual(FLEX_UI_VERSION);
+            // tslint:disable-next-line
```

Comment:
tslint:disable-next-line?

Reply:
Because tslint complains about using `dependencies.react` instead, but then if I do that, typescript complains saying `react` is not defined on `dependencies`

Reply:
o we're not using twilio-style here yet?